<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBWindow</class>
 <widget class="QWidget" name="DBWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>База данных</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/database.png</normaloff>../icons/database.png</iconset>
  </property>
  <widget class="QPushButton" name="CreateDB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string> Создать базу данных</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckConnection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить подключение
к базе данных</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="InformationBrowser">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectionSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройки подключения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
